src/csharp/Microsoft.Spark.Worker/Command/SqlCommandExecutor.cs

```diff
@@ -794,20 +794,33 @@ internal ArrowOrDataFrameGroupedMapCommandExecutor(Version version)
             {
                 FxDataFrame dataFrame = FxDataFrame.FromArrowRecordBatch(input);
                 FxDataFrame resultDataFrame = worker.Func(dataFrame);
+
                 IEnumerable<RecordBatch> recordBatches = resultDataFrame.ToArrowRecordBatches();
 
                 foreach (RecordBatch result in recordBatches)
                 {
-                    stat.NumEntriesProcessed += result.Length;
+                    ArrowBuffer.BitmapBuilder validityBitmapBuilder = new ArrowBuffer.BitmapBuilder();
```

Comment:
Just testing spark 3.0 CI at the moment. It should pass.

Once it does, we may need to exclude this code for versions < 3.0